### PR TITLE
create header

### PR DESCRIPTION
**Pull request by Noorin99**
**Create Header**
[
image
](url)
Relates #94

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+import { BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 import App from './App';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router>
+      <App />
+    </Router>
```
Comment (umarsalah) [inline client/src/index.jsx:11]: ???

Comment (Noorin99) [reply client/src/index.jsx:11]: We can use Router in the index file. For me it is actually a better practice than adding in in app

File: client/src/components/Header/index.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+              <Button
+                variant="contained"
+                className="register-button"
+                style={{
+                  backgroundColor: '#187F75',
+                  height: '31px',
+                  fontWeight: '500',
+                  width: '82%',
+                  textDecoration: 'none',
+                }}
```
Comment (ahmadfarid-059) [inline client/src/components/Header/index.jsx:69]: these buttons should not appear for the logged in users and should be replaced by the name of the user

File: client/src/components/Header/index.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import {
+  AppBar,
+  Box,
+  Toolbar,
+  Typography,
+  Button,
+  Stack,
+  Link,
+} from '@mui/material';
+import { Link as RouterLink } from 'react-router-dom';
```
Comment (ahmadfarid-059) [inline client/src/components/Header/index.jsx:10]: I think its readable when we use link!

File: client/src/components/index.jsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
+import Header from './Header';
 import EventsCard from './events-card';
 
-export default EventsCard;
+export default { Header, EventsCard };
```
Comment (ahmadfarid-059) [inline client/src/components/index.jsx:4]: default is not necessary because you export an object

**Review by ahmadfarid-059 — CHANGES_REQUESTED** [review]

File: client/src/index.jsx [hunk 5]
```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
 import './index.css';
 import App from './App';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router>
```
Comment (umarsalah) [inline client/src/index.jsx:9]:
```suggestion
       <App />
 ```
Just add the app in the strict mode tag without router we add it in the app itself

**Review by ahmadfarid-059 — APPROVED** [review]

**Review by umarsalah — APPROVED** [review]

**Pull request merged**